c_glib/arrow-glib/table.cpp

```diff
@@ -342,7 +342,25 @@ garrow_table_equal(GArrowTable *table, GArrowTable *other_table)
 {
   const auto arrow_table = garrow_table_get_raw(table);
   const auto arrow_other_table = garrow_table_get_raw(other_table);
-  return arrow_table->Equals(*arrow_other_table);
+  return arrow_table->Equals(*arrow_other_table, false);
```

Comment:
Could you add a note that this function doesn't compare metadata to document of this function?

Reply:
@shiro615 Sorry. I reconsider this.
Could you use `true` as the default? Because this change breaks backward compatibility. And I think that metadata should be checked too by default.

Reply:
Yes. I'll revert this change and fix test cases.

Reply:
I've fixed this.

Reply:
Sorry, some tests still failed. I'll fix them tonight.

Reply:
Fixed.